decidim/decidim
# Don't use a CTA for canceling a registration in a meeting

#### :tophat: What? Why?

After a participant registers in a Meeting, they have the "Cancel your registration" as a CTA. The thing is that we don't want participants to cancel their registrations, so it's better to show it in the secondary buttons in the sidebar. 

#### :pushpin: Related Issues

- Fixes #11302

#### Testing

1. Sign in as admin
2. Go to a meeting, enable registrations of the type "On this platform"
3. Go to the frontend, click on "Register"
4. See that the "Cancel your registration" button is in the sidebar

### :camera: Screenshots

#### Before
![CTA in the "Cancel your registration" button](https://github.com/decidim/decidim/assets/717367/ed6d677e-6470-491b-8117-9c10cf98d7e7)

#### After
![The cancel your registration button is in the secondary navigation](https://github.com/decidim/decidim/assets/717367/6a13fcd2-1c67-4113-ae74-f052f065654a)


:hearts: Thank you!
